Feature/visualizaciones 2

margarcac1 opened the pull request:
Para realizar la issue #1 necesito tener lo realizado en la rama feature/visualizaciones-2

margarcac1 committed:
MOSTRAR INFORMACIÓN EN TIEMPO REAL

Se ha implementado una modificación en Visualizaciones, concretamente, ahora se puede conocer información en tiempo real relativa al estado, censo total, porcentaje de participación y número de votos de una votación sin finalizar.
Para poder visualizar el cambio, iniciar una votación y acceder a /visualizer/idDeLaVotacion.

García Cáceres, María <@margarcac1>

margarcac1 committed:
MOSTRAR INFORMACION EN TIEMPO REAL

Se ha implementado la posibilidad de conocer la información relativa al censo, porcentaje de votantes,estado y número de votos de una votación si no ha finalizado.
Para ver la información se debe iniciar una votación y abir /visualizer/idVotacion

García Cáceres, María <@margarcac1>

raupargor reviewed (changes requested):
Buenas María, he detectado un bug en la feature que has estado desarrollando,
al acceder a una votación empezada sin votos, la página responde con un error 404.

A continuación voy a describir los pasos a seguir para recrear el error:
1. Me logueo como admin y creo una votación
2. Inicio la votación
3. Accedo al visualizer de la votación creada y empezada(visualizer/id)
4. Aparece una excepción 404 not found

Adjunto imagen del error detectado en un proximo mensaje, dado que github no me permite hacerlo aquí.

Creo que se puede solucionar añadiendo un if que compruebe esta casuística.

margarcac1 committed:
SOLUCION BUG VISUALIZACION EN TIEMPO REAL
El problema residía en que no se craba la página de visualización si todavía no había datos, es decir, si nadie había votado.

margarcac1 commented:
SOLUCION BUG VISUALIZACION EN TIEMPO REAL
El problema residía en que no se craba la página de visualización si todavía no había datos, es decir, si nadie había votado

raupargor reviewed (approved):
Todo parece funcionar correctamente, el error mencionado parece haber sido solucionado.
